lib/tty/logger/handlers/console.rb

```diff
@@ -118,7 +119,7 @@ def call(event)
             fmt << color.(style[:symbol])
             fmt << color.(style[:label]) + (" " * style[:levelpad])
           end
-          fmt << "%-25s" % event.message.join(" ")
+          fmt << "%-#{@message_padding}s" % event.message.join(" ")
```

Comment:
I considered extracting the whole format string, but that feels like a pretty sharp option to expose

Reply:
I actually like your idea of exposing the entire message format more than specifying padding. It increases flexibility and opens up more ways of formatting the message. As for the naming, let's go with `message_format`. There are already `date_format` and `time_format` configuration options. [lib/tty/logger/config.rb](https://github.com/piotrmurach/tty-logger/blob/master/lib/tty/logger/config.rb#L53-L54)